### ISSUE TITLE
Can the font be changed by the user?

### ISSUE THREAD

**Issue by DannyCarlton** [MakG10/jquery-meme-generator#13]
I'm trying to create a Bible verse meme generator and I'd like the user to be able to choose from several fonts. I've managed to substitute a Google font for the standard font used, but only as part of the initial settings. I still can't figure out how to change the fonts after the page is loaded, or how to set a different font for the second canvas layer.

**Comment by DannyCarlton** [comment]
Okay, here's the solution I found...

in jquery.memegenerator.js (which needs to be used instead of the min version for easy editing) just above the "PRIVATE METHOD" marker I added...

`		this.setFont = function(){
			MG.settings.defaultTextStyle.font=newfont;
		};`
...then in the html I create "newfont" as a global font (by creating it outside any function) and triggered the change with...
`$(".example.save img").memeGenerator("setFont");`

...of course after changing the variable "newfont" to the desired font. Once the font is changed the text needs to be redrawn which can be done with trigger("change") of the input for that textbox.

**Comment by MakG10** [comment]
Yeah, I guess you have to use this workaround for now. I will add an "option" method to manually change any of the initial settings.
